src/pages/about-us.tsx

```diff
@@ -9,13 +10,61 @@ import styles from './contentPage.module.scss';
 import NextSeoWrapper from '@/components/NextSeoWrapper';
 import PageContainer from '@/components/PageContainer';
 import Link from '@/dls/Link/Link';
+import ChaptersData from '@/types/ChaptersData';
+import { getAllChaptersData } from '@/utils/chapter';
 import { logTarteelLinkClick } from '@/utils/eventLogger';
 import { getBlurDataUrl } from '@/utils/image';
 import { getLanguageAlternates } from '@/utils/locale';
 import { getCanonicalUrl } from '@/utils/navigation';
 
-const path = '/about-us';
-const AboutUsPage = () => {
+const PATH = '/about-us';
+const CREDIT_LINKS = [
+  {
+    key: 0,
+    i18nKey: 'about:credits.tanzil',
+    href: 'https://tanzil.net/',
+  },
+  {
+    key: 1,
+    i18nKey: 'about:credits.quran-complex',
+    href: 'https://qurancomplex.gov.sa/',
+  },
+  {
+    key: 2,
+    i18nKey: 'about:credits.quran-align',
+    href: 'https://github.com/cpfair/quran-align',
+  },
+  {
+    key: 3,
+    i18nKey: 'about:credits.quran-enc',
+    href: 'https://quranenc.com/en/home',
+  },
+  {
+    key: 4,
+    i18nKey: 'about:credits.zekr',
+    href: 'https://zekr.org',
+  },
+  {
+    key: 5,
+    i18nKey: 'about:credits.lokalize',
+    href: 'https://lokalise.com/',
+    image: '/images/lokalize.png',
+    alt: 'Lokalise',
+  },
+  {
+    key: 6,
+    i18nKey: 'about:credits.vercel',
+    href: 'https://vercel.com/?utm_source=quran-pro&utm_campaign=oss',
+    image: '/images/powered-by-vercel.svg',
+    alt: 'Vercel',
+  },
+];
```

Comment:
This change is out of the scope of the PR, right? 🤔 

I would leave refactoring work in a separate issue and just narrow down the changes to what's required in the PR 😄